actions/views/threads.ts

```diff
@@ -41,14 +41,11 @@ export function manuallyMarkThreadAsUnread(threadId: string, lastViewedAt: numbe
     ]);
 }
 
-export function switchToGlobalThreads() {
-    return (_dispatch: DispatchFunc, getState: GetStateFunc) => {
-        const state = getState() as GlobalState;
-        const teamUrl = getCurrentRelativeTeamUrl(state);
-        browserHistory.push(`${teamUrl}/threads`);
+export function switchToGlobalThreads(state: GlobalState) {
```

Comment:
```suggestion
export function switchToGlobalThreads() {
    return (_dispatch: DispatchFunc, getState: GetStateFunc) => {
    // get the state from here
```

Reply:
This was the original implementation before I made any changes. 
Can you please check this comment [https://github.com/mattermost/mattermost-webapp/pull/11331#issuecomment-1277636398](https://github.com/mattermost/mattermost-webapp/pull/11331#issuecomment-1277636398)?